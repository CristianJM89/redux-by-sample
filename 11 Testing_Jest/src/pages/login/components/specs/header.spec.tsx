import * as React from 'react';
const renderer: any = require('react-test-renderer');
import {Header} from '../header';

describe('header', () => {
  describe('#render', () => {
    it('renders as expected', () => {
      //Arrange

      //Act
      const tree = renderer.create(
        <Header />
      ).toJSON();

      //Assert
      expect(tree).toMatchSnapshot();
    });
  });
});
