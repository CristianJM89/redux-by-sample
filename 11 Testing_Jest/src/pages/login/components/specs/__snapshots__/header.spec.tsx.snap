// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders as expected 1`] = `
<div
  className="panel-heading"
>
  <h3
    className="panel-title"
  >
    Please sign in
  </h3>
</div>
`;
